website/pages/en/substreams/[[...slug]].mdx

```diff
@@ -18,24 +18,38 @@ export async function getStaticPaths() {
 export async function getStaticProps({ params: { slug = ['README'] } }) {
   const baseURL = 'https://raw.githubusercontent.com/streamingfast/substreams/develop/docs/'
   const paths = slug.join('/')
-  let response = await fetch(`${baseURL}${paths}.md`)
+  let fileURL = `${paths}.md`
+  let response = await fetch(baseURL + fileURL)
   if (response.status === 404 && paths !== 'README') {
-    response = await fetch(`${baseURL}${paths}/README.md`)
+    fileURL = `$${paths}/README.md`
+    response = await fetch(baseURL + fileURL)
   }
-  const data = await response.text()
+  const data = (await response.text())
+    // replace {% embed ... %} with <iframe />
+    .replaceAll(
+      /{%\s+embed\s+url="(.*?)"\s+%}/g,
+      (...m) =>
+        `<iframe src="${m[1].replace(
+          // we need enhance YouTube links, otherwise they will be not loaded in iframe
+          'youtube.com/watch?v=',
+          'youtube.com/embed/'
+        )}" style={{aspectRatio: 16/9, width: '100%'}}/>`
+    )
+    // remove gitbook {% ... %} elements
+    .replaceAll(/{%.*?%}/g, '')
+    // close img tags only if he doesn't point to .gitbook
+    .replaceAll(/<img(.*?)>/g, (...m) => (m[1].includes('src=".gitbook') ? '' : `<img${m[1]}/>`))
+    // fixes http://localhost:3000/en/substreams/reference-and-specs/authentication/
+    .replaceAll('<pre class="language-bash" data-overflow="wrap"><code class="lang-bash">', '```bash\n')
+    // fixes http://localhost:3000/en/substreams/developers-guide/modules/inputs/
+    .replaceAll('<pre class="language-yaml" data-title="manifest excerpt"><code class="lang-yaml">', '```yaml\n')
+    .replaceAll('</code></pre>', '```')
   const mdx = await buildDynamicMDX(data, {
     mdxOptions: {
-      format: 'md',
+      format: paths.endsWith('/change-log') ? 'md' : 'mdx',
```

Comment:
```suggestion
      // change-log contains `{variable}` text that is thrown an error by MDX2 parser since he treat it as variable injection, to fix it we parse chang-log with the markdown parser
      format: paths.endsWith('/change-log') ? 'md' : 'mdx',
```